.github/workflows/haskell.yml

```diff
@@ -157,20 +159,21 @@ jobs:
         git diff --exit-code
 
   # Check formatting for cabal files
-  cabal-fmt:
+  cabal-gild:
```

Comment:
I like `cabal-fmt`, I'm not sure it's necessary to switch

Reply:
It's less reader friendly -- we're not changing these cabals that often that the diff matters

Reply:
Ok. Reverting to cabal-fmt